pybossa/view/projects.py

```diff
@@ -3169,6 +3171,8 @@ def ext_config(short_name):
                     flash(gettext('Please correct the errors', 'error'))
                 ext_conf[form_name] = form.data
                 ext_conf[form_name].pop('csrf_token', None)     #Fflask-wtf v0.14.2 issue 102
+                if not ext_conf[form_name].get('bpv_key_id'):
+                    ext_conf[form_name].pop('bpv_key_id', None)
```

Comment:
if condition not required.

Reply:
I think we only pop out the `bpv_key_id ` field when the value is `""`.

Reply:
ok